doc/API.md

```diff
@@ -82,104 +82,61 @@ Table of contents
 Daemon
 ------
 
-Queries:
-
-* /daemon/constants [GET]
-* /daemon/stop      [GET]
-* /daemon/version   [GET]
+| Route                                     | HTTP verb |
+| ----------------------------------------- | --------- |
+| [/daemon/constants](#daemonconstants-get) | GET       |
+| [/daemon/stop](#daemonstop-get)           | GET       |
+| [/daemon/version](#daemonversion-get)     | GET       |
 
 #### /daemon/constants [GET]
 
-Function: Returns the set of constants in use.
+returns the set of constants in use.
 
-Parameters: none
-
-Response:
-```
-struct {
-	genesistimestamp      types.Timestamp (uint64)
-	blocksizelimit        uint64
-	blockfrequency        types.BlockHeight (uint64)
-	targetwindow          types.BlockHeight (uint64)
-	mediantimestampwindow uint64
-	futurethreshold       types.Timestamp   (uint64)
-	siafundcount          types.Currency    (string)
-	siafundportion        *big.Rat          (string)
-	maturitydelay         types.BlockHeight (uint64)
-
-	initialcoinbase uint64
-	minimumcoinbase uint64
-
-	roottarget types.Target (byte array)
-	rootdepth  types.Target (byte array)
-
-	maxadjustmentup   *big.Rat (string)
-	maxadjustmentdown *big.Rat (string)
-
-	siacoinprecision types.Currency (string)
+###### JSON Response [(with comments)](/doc/api/Daemon.md#json-response)
+```javascript
+{
+  "genesistimestamp":       1257894000, // Unix time
+  "blocksizelimit":         2000000,    // bytes
+  "blockfrequency":         600,        // seconds per block
+  "targetwindow":           1000,       // blocks
+  "mediantimestampwindow":  11,         // TODO: What are the units? blocks?
+  "futurethreshold":        10800,      // seconds
+  "extremefuturethreshold": 21600,      // seconds
+  "siafundcount":           "10000",
+  "siafundportion":         "39/1000",
+  "maturitydelay":          144,        // blocks
+
+  "initialcoinbase": 300000, // TODO: What are the units? SiaCoins? But not hastings? This is probably the only place we use a uint64 for currency and don't use hastings.
```

Comment:
Siacoins. Definitely breaks from the convention of returning everything in hastings, but we have a compatibility promise that we need to uphold. So, I'd add some extra documentation explaining that, while this is presenting in siacoins and not hastings, it is unusual and doesn't follow the rest of the API.